npc/re/jobs/2e/rebellion.txt

```diff
@@ -0,0 +1,1093 @@
+		next;
+		mes("^0000ffElwin see you turn into another form and she is very angry. Wait for your current form ends, and then find her again.^000000");
+		next;
+		if (select("Wait.", "Go away.") == 1) {
+			mes("[Elwin Conick]");
+			mes("It's not worth to give up, now go out and relax..");
+			close;
+		}
+		mes("[Elwin Conick]");
+		mes("Come back when you aren't in any form.");
+		close2;
+		warp("job_gun", 81, 29);
+		end;
+	} else {
+		mes("[Elwin Conick]");
+		mes("Time over. You failed the test. Let me take all your leftover test materials.");
+		.@st_job1 = countitem(Steel_Article);
+		.@st_job2 = countitem(Steel_Article_);
+		if (.@st_job1 > 0) {
+			delitem(Steel_Article, .@st_job1);
+		}
+		if (.@st_job2 > 0) {
+			delitem(Steel_Article_, .@st_job2);
+		}
+		if (countitem(Tiny_Flame) > 0) {
+			delitem(Tiny_Flame, 1);
+		}
+		next;
+		mes("[Elwin Conick]");
+		mes("Let's start again.");
+		close2;
+		warp("job_gun", 117, 41);
+		end;
+	}
+	end;
+}
+
+job_gun,120,138,3	script	Ivan Sidorenko#1	4_M_REBELLION,2,2,{
+	if (Class != Job_Gunslinger) {
+		mes("[Ivan Sidorenko]");
+		mes("This is shooting test area, you can't come here as you like.");
+		close2;
+		warp("job_gun", 81, 29);
+		end;
+	}
+	if (questprogress(12342) < 2) {
+		mes("[Ivan Sidorenko]");
+		mes("There seems to be a mistake, you can't stay here.");
+		close2;
+		warp("job_gun", 81, 29);
+		end;
+	}
+	if (questprogress(12343) < 1) {
+		mes("[Ivan Sidorenko]");
+		mes("Hey, stay focused--we're using live ammo here, and you can get shot anytime.");
+		next;
+		mes("[Ivan Sidorenko]");
+		mes("Let me make this quick. Basically you'll be running in this shooting range consisting of 5 sections.");
+		next;
+		mes("[Ivan Sidorenko]");
+		mes("The objective is to find enemies in the midst of a crowd quickly and without fail.");
+		next;
+		mes("[Ivan Sidorenko]");
+		mes("Among all the targets, you must shoot only the ^0000ffStandard-issue Targets^000000. If you make a mistake, you'll lose points.");
+		next;
+		mes("[Ivan Sidorenko]");
+		mes("Let's try. There aren't many ^0000ffStandard-issue Targets^000000, so keep your eyes peeled.");
+		next;
+		mes("[Ivan Sidorenko]");
+		mes("^0000ffYou have 3 minutes to finish this test. If you can't finish before your transformation expires^000000, you'll fail. If that happens, just follow the path back to me.");
+		next;
+		mes("[Ivan Sidorenko]");
+		mes("^0000ffYou'll fail if you miss a Standard-issue Target, or shoot 3 incorrect targets^000000. Let's get started.");
+		setquest(12343);
+		setquest(12344);
+		montransform(G_L_SHECIL, 180000);
+		close2;
+		warp("job_gun", 102, 146);
+		end;
+	} else if (questprogress(12343) == 1) {
+		.@standard_target = 0;
+		.@failed_target = 0;
+		.@total_target = 0;
+		if (questprogress(12343, HUNTING) == 2) {
+			.@standard_target = 10;
+		}
+		if (questprogress(12344, HUNTING) == 2) {
+			.@failed_target = 2;
+		}
+		if (getstatus(SC_MONSTER_TRANSFORM, 1) != 2431) {
```

Comment:
```suggestion
		if (getstatus(SC_MONSTER_TRANSFORM, 1) != G_L_SHECIL) {
```

Reply:
done, will be pushed together in next commit